packages/calcite-components/src/components/card/card.tsx

```diff
@@ -156,30 +231,56 @@ export class Card implements ConditionalSlotComponent, LocalizedComponent, T9nCo
 
   @State() defaultMessages: CardMessages;
 
+  @State() private hasContent = false;
+
   //--------------------------------------------------------------------------
   //
   //  Private Methods
   //
   //--------------------------------------------------------------------------
 
-  private cardSelectClick = (): void => {
-    this.selectCard();
+  private handleDefaultSlotChange = (event: Event): void => {
+    this.hasContent = slotChangeHasAssignedElement(event);
   };
 
-  private cardSelectKeyDown = (event: KeyboardEvent): void => {
+  // to be removed in future release
```

Comment:
IMO, this comment isn't needed since the render method conveys that it will be removed later on. Applies to similar comments.